### PR TITLE
Update package-security.asciidoc

### PR DESCRIPTION
**Pull request by CantaTronic**
Add a link to password reset in case the user has no access to the auto-generated password

### CODE REVIEW

File: docs/reference/setup/install/package-security.asciidoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ the `elastic` built-in superuser.
 * Certificates and keys for TLS are generated for the transport and HTTP layer,
 and TLS is enabled and configured with these keys and certificates.
 
-The password and certificate and keys are output to your terminal.
+The password and certificate and keys are output to your terminal. If that's not the case, reset the password for the user `elastic` with <<reset-password,`elasticsearch-reset-password`>>.
```
Comment (leemthompo) [inline docs/reference/setup/install/package-security.asciidoc:13]:
```suggestion
The password and certificate and keys are output to your terminal.
You can reset the password for the user `elastic` with the <<reset-password,`elasticsearch-reset-password`>> command.
```

**Comment by leemthompo** [comment]
Thanks @CantaTronic, this is a helpful link! 

You'll need to sign the [contributor agreement](https://github.com/elastic/elasticsearch/pull/103492#issuecomment-1858841643) so we can merge this.

We'll also want to target `main` and then backport the change to `8.11`. I can take care of that.

**Comment by andrei-verdes** [comment]
@leemthompo created a new PR here to get this change through: https://github.com/elastic/elasticsearch/pull/109314. (I got this on CodeTriage, trying to clear up issues/PRs that are left inactive)

**Comment by andrei-verdes** [comment]
@leemthompo this can now be closed, it was merged here: https://github.com/elastic/elasticsearch/pull/109314 . Thanks :)

**Comment by leemthompo** [comment]
Superseded by https://github.com/elastic/elasticsearch/pull/109314

**Pull request closed without merging**